drivers/clock_control/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if (CLOCK_CONTROL_NRF_LL_NRFXLIB)
```

Comment:
This isn't a Kconfig option?

Reply:
Isn't it? See `drivers/clock_control/Kconfig`

Reply:
s/CLOCK_CONTROL_NRF_LL_NRFXLIB/CONFIG_CLOCK_CONTROL_NRF_LL_NRFXLIB/

Reply:
- [ ] s/CLOCK_CONTROL_NRF_LL_NRFXLIB/CONFIG_CLOCK_CONTROL_NRF_LL_NRFXLIB/